erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -1172,3 +1172,14 @@ def update_item(source_doc, target_doc, source_parent):
 	}, target_doc, set_missing_values)
 
 	return doclist
+
+
+@frappe.whitelist()
+def init_print_check(values, selected_docs, doctype):
+	docs = json.loads(selected_docs)
+	for doc in docs:
+		frappe.db.set_value(doctype, doc.get("name"), {
+			"reference_no": values,
+			"reference_date": nowdate()
+		})
+		values = int(values)+1
```

Comment:
Change this to an integer before the loop, so you don't have to keep calling `int`